parl/core/fluid/agent.py

```diff
@@ -26,11 +26,48 @@
 
 
 class Agent(AgentBase):
+    """
+    | `alias`: ``parl.Agent``
+    | `alias`: ``parl.core.fluid.agent.Agent``
+
+    | Agent is one of three basic classes of PARL. 
+
+    | It is responsible for interacting with the environment and collecting data for training the policy.
+    | To implement a customized ``Agent``, users can:
+
+      .. code-block:: python
+
+        import parl
+
+        class MyAgent(parl.Agent):
+            def __init__(self, algorithm, act_dim):
+                super(MyAgent, self).__init__(algorithm)
+                self.act_dim = act_dim
+      This class will automatically initialize all the parameters of neural network, and provides an executor for users to run programs(self.fluid_executor).
```

Comment:
This class will initialize the neural network parameters automatically, and provides an executor for users to run the programs (self.fluid_executor).